### PR TITLE
Added SandBoxDemo for SingleDropdownList

### PR DESCRIPTION
**Pull request by divyanshu-rawat**
Here is the link to the Demo - https://codesandbox.io/s/m9j1k1rq9p

### CODE REVIEW

**Comment by divyanshu-rawat** [comment]
@bietkul, @metagrover  please review this 👍

File: content/docs/list-components/SingleDropdownList.md [hunk 1]
```diff
@@ -119,6 +119,12 @@ or
 - **showLoadMore** `Boolean` [optional]  
     defaults to `false` and works only with elasticsearch >= 6 since it uses [composite aggregations](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-composite-aggregation.html). This adds a "Load More" button to load the aggs on demand combined with the `size` prop. Composite aggregations are in beta and this is an experimental API which might change in a future release.
 
+## Demo
+
+<br />
+
+<iframe src="https://codesandbox.io/s/m9j1k1rq9p" style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;" sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"></iframe>
+
```
Comment (bietkul) [inline content/docs/list-components/SingleDropdownList.md:127]: @divyanshu-rawat Please add SelectedFilters in all codesanbox demos.
And keep the same layout in full screen mode as we have in storybook demos( in this example dropdown list covers the results ).

**Review by bietkul — COMMENTED** [review]
@divyanshu-rawat Please add SelectedFilters in all codesanbox demos.
And keep the same layout in full screen mode as we have in storybook demos( in this example dropdown list covers the results )

**Comment by divyanshu-rawat** [comment]
@bietkul Updated the PR 👍

**Comment by bietkul** [comment]
I'm not able to find the changes in codesandbox demo.

**Review by bietkul — CHANGES_REQUESTED** [review]
You need to update the nav.yml so we can access the demos from sidebar.

**Comment by divyanshu-rawat** [comment]
@bietkul sandbox updated - https://codesandbox.io/s/ly6w2k7y27
it wasn't saved.

whats is `nav.yml`?

**Comment by jyash97** [comment]
`nav.yml` is file used to define the items and their navigation that are present on the right sidebar.

check this [file](https://github.com/appbaseio/reactive-manual/blob/v1-vue/content/docs/list-components/nav.yml)
On line 16 we have `demo` item and its link i.e href attribute.

**Pull request merged**